tests/playwright/src/model/pages/resources-page.ts

```
@@ -30,21 +30,22 @@ export class ResourcesPage extends SettingsPage {
   }
 
-  public async resourceCardIsVisible(resourceLabel: string): Promise<boolean> {
-    return (await this.resourceCardLocatorGenerator(resourceLabel).count()) > 0;
+  public async resourceCardIsVisible(resourceCardAriaLabel: string): Promise<boolean> {
+    return (await this.resourceCardLocatorGenerator(resourceCardAriaLabel).count()) > 0;
   }
 
-  public async goToCreateNewResourcePage(resourceLabel: string): Promise<void> {
-    if (!(await this.resourceCardIsVisible(resourceLabel))) {
-      throw new Error(`Resource card with label ${resourceLabel} is not available`);
+  public async goToCreateNewResourcePage(resourceCardAriaLabel: string, newResourceName: string = ''): Promise<void> {
```

Biome (2.1.2)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)